dashboard/apps/moj_admin/static/moj_admin/js/dwp_admin.js

```diff
@@ -0,0 +1,38 @@
+    $(document).ready(function () {
+      // Get the URL
+      var url = window.location.href.split('/').pop();
+
+      if (url === '') {
+        url = '/';
+      }
+
+      // Check the href of each link in the sidebar
+      $('.admin-sidebar-content a').each(function () {
+        if (url === $(this).attr('href')) {
+          // If the href matches the current URL set it as active
+          $(this).parents('li').addClass('active');
+        }
+      });
+    });
+    //
+    // // Tab Panes
```

Comment:
It isn't now, but would be to use some of the styling on the DWP admin. Suggest leaving in until Maria has had a look once I've made some of the other changes.